src/renderer/splash.tsx

```diff
@@ -16,81 +20,83 @@ const Splash = memo(() => {
         ipcRenderer.on('checking-port', () => {
             setShowProgressBar(false);
             setOverallProgressPercentage(0.1);
-            setStatus('Checking for available port...');
+            setStatus(`${t('CHECKING_PORT', 'Checking for available port')}...`);
```

Comment:
Those are not just fallback strings, they're defaults. They get automatically put into the keys via the Babel plugin so we don't have to actually manage the English localization. Without this I'd never know what the actual string says when looking at code, and we'd have to manually keep changing things in the English locale. Plus i think this would be a fallback for any language that doesn't have a particular key. I18next recommends you do this, at least instead of using the full string as the key (which i initially wanted to do)

Reply:
Good point.